services/supervisord/logs.go

```diff
@@ -214,6 +215,30 @@ func (l *Logs) files(ctx context.Context) []fileContent {
 		Err:  err,
 	})
 
+	// add pprof
+	if pprofConfig != nil {
+		traceBytes, err := pprofUtils.Trace(pprofConfig.TraceDuration)
+		files = append(files, fileContent{
+			Name: "pprof/trace.out",
+			Data: traceBytes,
+			Err:  err,
+		})
+
+		profileBytes, err := pprofUtils.Profile(pprofConfig.ProfileDuration)
+		files = append(files, fileContent{
+			Name: "pprof/profile.pb.gz",
+			Data: profileBytes,
+			Err:  err,
+		})
```

Comment:
can we run them in parallel to speed up?

Reply:
Sure. Fixed.